### ISSUE TITLE
Waterfox display wrong close buttons and right menu style on Windows [BUG]

### ISSUE THREAD

**Issue by justrico** [AdamXweb/WhiteSurFirefoxThemeMacOS#62]
**Describe the bug**
1. The Waterfox close buttons display strange icons.
2. The right menu text display black color in dark mode, so that I even can't see the text.

**To Reproduce**
1. Install [Waterfox](https://www.waterfox.net/download) G4.0.7
3. Install WhiteSur theme by follow [README.md](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/blob/master/README.md#manual-installation-macos--windows) file
4. Restart Waterfox and see error

**Expected behavior**
1. Perhaps display the correct icons like the picture you showed in README.md
2. The right menu text display white color to see the text clearly
3. Could you please make more blank area in the top menu bar? Everytime I move the window had to hardly find the small blank area between icons. Thanks.

**Screenshots**
![图片](https://user-images.githubusercontent.com/52879214/156195758-121f685e-0187-405c-8c5e-7f06c48aeb10.png)

**Desktop (please complete the following information):**
 - OS: Windows 11 21H2 22000.434
 - Waterfox version: G4.0.7
 - WhiteSur theme: 1.4.1
 - Any options added during installation: None

**Additional context**

**Comment by AdamXweb** [comment]
Looking into Waterfox, their latest release [G4.0.7](https://github.com/WaterfoxCo/Waterfox/releases) may be based on an older version of Firefox.
Not sure if i'd be able to support it if it deviates too far from the standard Firefox branch.

Good suggestion on the padding between the URL bar. May be time for a rework.
Could you test on the standard version of firefox, as I haven't tested on Windows 11 just yet.

**Comment by justrico** [comment]
> Looking into Waterfox, their latest release [G4.0.7](https://github.com/WaterfoxCo/Waterfox/releases) may be based on an older version of Firefox. Not sure if i'd be able to support it if it deviates too far from the standard Firefox branch.
> 
> Good suggestion on the padding between the URL bar. May be time for a rework. Could you test on the standard version of firefox, as I haven't tested on Windows 11 just yet.

Yes. I test on the latest standard version of Firefox(97.0.2) on Windows 11. It looks very nice.
![图片](https://user-images.githubusercontent.com/52879214/156917327-1e991b98-6215-4537-8f04-693baabb3e05.png)

I checked both kernel. Waterfox based on old 91.0 version. Not so old. I think waterfox team may has changed something in core style code so that makes buttons and menu texts problems.
![图片](https://user-images.githubusercontent.com/52879214/156917665-0b049810-dd62-4b07-8947-fec8aac3b7dc.png)

**Comment by AdamXweb** [comment]
You may be able to try a previous release of the theme. Prior to Proton being the main styling, the theme had [supported v91 ](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/releases/tag/v1.3.1) of Firefox.

**Comment by justrico** [comment]
> You may be able to try a previous release of the theme. Prior to Proton being the main styling, the theme had [supported v91 ](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/releases/tag/v1.3.1) of Firefox.

I tried [WhiteSurFirefoxThemeMacOS v1.3.1](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/releases/tag/v1.3.1), right menu background color changed to light color, not as normally color displayed in dark mode. But it is clearly to read words now :-P

![图片](https://user-images.githubusercontent.com/52879214/156929552-bf7905dd-2628-4633-9af6-7ceed43d13a4.png)

Buttons still have problem, but it doesn't matter, I decided to work back on Firefox. Thanks for your help and reply. Have a good day :-)
